editor/src/components/inspector/inspector.spec.browser2.tsx

```diff
@@ -0,0 +1,99 @@
+import { r } from 'tar'
+import { elementPath } from '../../core/shared/element-path'
+import { mouseClickAtPoint } from '../canvas/event-helpers.test-utils'
+import { getPrintedUiJsCode, renderTestEditorWithCode } from '../canvas/ui-jsx.test-utils'
+import { selectComponents } from '../editor/actions/action-creators'
+import { AspectRatioLockButtonTestId } from './sections/layout-section/self-layout-subsection/gigantic-size-pins-subsection'
+
+describe('inspector', () => {
+  it('toggle aspect ratio lock off', async () => {
+    const codeAfterToggle = await runToggleAspectRatioLockTest(true)
+    expect(codeAfterToggle).toEqual(`import * as React from 'react'
+import { Scene, Storyboard } from 'utopia-api'
+
+export var storyboard = (
+  <Storyboard data-uid='0cd'>
+    <div
+      style={{
+        backgroundColor: '#0091FFAA',
+        position: 'absolute',
+        left: 33,
+        top: 307,
+        width: 363,
+        height: 426,
+      }}
+      data-uid='7a0'
+    />
+  </Storyboard>
+)
+`)
+  })
+
+  it('toggle aspect ratio lock on', async () => {
+    const codeAfterToggle = await runToggleAspectRatioLockTest(false)
```

Comment:
please make `aspectRatioLocked` a string literal union, to improve the readability here

Reply:
thanks, fixed!